src/crackmapexec.yml

```diff
@@ -0,0 +1,18 @@
+---
+# The CrackMapExec dependency impacket now requires the krb5-config
```

Comment:
This is a [dependency for CrackMapExec](https://github.com/byt3bl33d3r/CrackMapExec/blob/593062ecfb4de28f21397862928b66f0eba5625f/requirements.txt#L28) and I'm not sure what changed but the [line that fails](https://github.com/pythongssapi/python-gssapi/blob/08e2b09fc68082fbec7b94aadb8c2c36efaea1b0/setup.py#L128) is not new.

```suggestion
# The CrackMapExec dependency gssapi requires the krb5-config
```